webui/react/src/components/TupleRow.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import css from './TupleRow.module.scss';
+
+interface Props {
+  label: string;
+  children?: React.ReactNode;
+}
+
+const TupleRow: React.FC<Props> = ({ label, children }: Props) => {
+
+  if (children === undefined) return <></>;
+  return (
+    <tr className={css.base} key={label}>
+      <td className={css.label}>{label}</td>
+      <td>
+        {[ 'string', 'number' ].includes(typeof children) ?
+          <span>{children}</span> : children
+        }
+      </td>
+    </tr>
+  );
+};
+
+export default TupleRow;
```

Comment:
**suggestion:** The `TupleRow` doesn't quite stand on its own as a component just because it can't render on it's own and requires `table` and `tbody` to render. Instead, this is a good opportunity to build a `InfoBox` component with a separate `Row` subcomponent within that component.

Reply:
good point. what do you think is a good interface for this infobox component? how about a list of tuples?

Reply:
How about...
```
{ label: string, info?: React.ReactNode }[];
```
And if the `info` field is not specified, it will skip rendering it.